src/lib/components/CmsForm.svelte

```diff
@@ -6,6 +6,8 @@
 		TINYMCE_TOOLBAR
 	} from '../../routes/(app)/admin[[hash=admin_hash]]/cms/tinymce-plugins';
 	import { MAX_CONTENT_LIMIT } from '$lib/types/CmsPage';
+	import { generateId } from '$lib/utils/generateId';
+	import { z } from 'zod';
```

Comment:
Let's not add this dependency on the frontend (and increase bundle size), it's only used in the backend